modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,344 @@
+= MongoDB - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Upgrade Anypoint Connector for MongoDB (MongoDB Connector) to version 6.0.0.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|5.3.x |6.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Changed operation names, metadata, fields, and values
++
+*Reviewers, what values were changed? I do not see them in the PR. Also, were some operations added besides the Object Listener input source?*
++
+* Changed input sources
+* Changed connection types
+
+=== Changed Operation Names
+
+The following operation names changed in MongoDB 6.0.0:
+
+[%header]
+|===
+|5.3.x|6.0.0
+|Map reduce objects |Map reduce
+|Exists collection |Collection exists
+|List indices |List indexes
+|===
+
+== Changed Operations Parameters
+
+[%header,cols="19%,15%,15%,19%,15%,15%"]
+|===
+3+|Existing in MongoDB 5.x 3+| Modified in MongoDB 6.x
+|Operation Name | Input Params  |Output Params  | Operation Name                             |Input Params                                      |Output Params
```

Comment:
Please fill the changes related to the outputs because several outputs change, like insert documentS output